compile-angel.el

```diff
@@ -479,13 +486,18 @@ FEATURE and FILENAME are the same arguments as the `require' function."
              "compile-angel--advice-before-require %s (%s)")
      el-file (type-of el-file))))
 
-(defun compile-angel-compile-features ()
-  "Compile all loaded features that are in the `features' variable."
-  (let ((compile-angel--native-compile-when-jit-enabled t))
-    (dolist (feature features)
-      (compile-angel--debug-message
-       "compile-angel-compile-features: %s" feature)
-      (compile-angel--entry-point nil feature))))
+(defun compile-angel-compile-loaded ()
+  "Compile all previously loaded files."
+  (let ((compile-angel--native-compile-when-jit-enabled t)
+        (exts (compile-angel--el-file-extensions)))
+    (dolist (entry load-history)
+      (let ((fname (car entry)))
+        (if (member (file-name-extension fname t) exts)
```

Comment:
You can use the `compile-angel--is-el-file` function, which relies on `compile-angel--el-file-regexp`, dynamically updated whenever `load-file-rep-suffixes` is modified.